website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -163,6 +163,35 @@ If no filters have been applied, use a [Tooltip](/components/tooltip) coupled wi
 
 ![Empty state in applied filters](/assets/patterns/filter-patterns/applied-filters-empty-state.png =450x*)
 
+### Avoiding empty state
+
+Because filtering is often reductive in nature and driven by a user action, it might not always be possible to avoid an empty state within a data set. This can be helped by leveraging specific component variants and properties, and by taking steps to prevent mistakes or errors that would result in an empty state prior to them occurring.
+
+#### ListItem count
+
+Use the `count` property within the Dropdown ListItem to more clearly explicitly call attention to filters that will return zero results.
```

Comment:
Reasons why this is not always feasible:
- there can be very strong limitations in doing this (costly operation on the backend or frontend)
- the number could be very high (hundreds of thousands?) so it would not fit or convey no information per se

```suggestion
If technically possible, use the `count` property within the Dropdown ListItem to more clearly explicitly call attention to filters that will return zero results.
```

Reply:
I'd also consider dropping "clearly" or "explicitly". I don't think both are needed.

Reply:
Ah, good point on the technical side.